h/streamer/filter.py

```diff
@@ -34,8 +32,20 @@ class FilterHandler:
     def __init__(self, filter_json):
         self.filter = filter_json
 
-    def evaluate_clause(self, clause, target):
-        field_value = resolve_pointer(target, clause["field"], None)
+    def evaluate_clause(self, clause, annotation):
+        field_path = clause["field"]
+
+        # Extract the annotation property that corresponds to the "field"
+        # path in the filter. This only supports a small set of paths
+        # which are actually used by the Hypothesis client.
```

Comment:
The Hypothesis client configures one of two types of filter. See `load-annotations.js` in the client:

- Notify the client about all annotations on a particular URL. This is used in the sidebar.
- Notify the client about all updates to a specific annotation or its replies. This is used on the single annotation page.

Reply:
I can't for some reason parse this comment. What paths?